src/components/post/MakePost.jsx

```diff
@@ -41,6 +41,9 @@ class MakePost extends Component {
     originalPost.visibility = postVisibility;
 
     onSubmit(originalPost);
+    this.setState({
+      postContent:"",
```

Comment:
we need to reset the image too .. just in case the TAs try to create a post with an image - right now we have no way to get rid of the image after you upload it lol